app/controllers/taxons_controller.rb

```diff
@@ -16,6 +16,17 @@ def create
     redirect_to taxons_path
   end
 
+  def show
+    @taxon = TaxonForm.build(content_id: params[:id])
+    # FIXME: ideally the gds-api-adapters should be updated to include support
```

Comment:
I think it's best to update gds-api-adapters before merging this, so that we don't have to come back to update this app.